### ISSUE TITLE
Is it possible to load outer ephemeral public from uncompressed point format?

### ISSUE THREAD

**Issue by mvinnik92** [randombit/botan#1514]
Hello!

Botan::DataSource_Memory ds_ephPublic("BHDIc7M07hAhonEm27r6NqfzPkcQsIydFrt/h+nZoe7TC647EJzAUJBuL23q8B1wLJn925oXkupOU9p8e3wXl+g\\u003d");
Format: byte for type, 32 bytes for x, 32 bytes for y.

Is it loadable? Small amount of examples found on it so asking

**Comment by randombit** [comment]
Yes, use OS2ECP function in point_gfp.h. It requires knowing what curve is associated with the key since that is not included in the serialization.

**Comment by mvinnik92** [comment]
Thanks a lot!
